<launch>
  <group ns="uav1">
    <param name="robot_name" value="uav1" />
    <!-- <param name="robot_id" value="1" /> -->

    <!-- static transform from odom to map-->
    <!-- <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0 0 0 0 0 0 map drone_odom 0" /> -->

    <!-- pcl_obstacle_avoidance launch file -->
    <include file="$(find dekf_sensor_fusion)/launch/dekf.launch"/>

  </group>

</launch>
